docs/esp8266/tutorial/repl.rst

```diff
@@ -144,9 +144,9 @@ function::
 
     >>> def toggle(p):
     ...    p.value(not p.value())
-    ...    
-    ...    
-    ...    
```

Comment:
As above, it's correct to have spaces here because that's what is output at the prompt.